usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3276,6 +3281,7 @@ CK_RV token_specific_set_attrs_for_new_object(STDLL_TokData_t *tokdata,
  *                     End of CCA protected key option
  *
  ******************************************************************************/
```

Comment:
How about the PKEY config option if NO_PKEY is defined? I would say we should still support it, but only accept the DISABLED setting. Also, in this case the default of course is DISABLED.

Also add a note to the config file description that protected keys are only available on the s390x (Linux on IBM Z) platform

Reply:
Yes, I didn't put reading the pkey_mode option out of the config file into a #ifndef NO_PKEY block, so the option is read in any case. But as all functions that evaluate 'cca_data->pkey_mode' are enclosed in #ifndef NO_PKEY it doesn't matter what the value is.  But I can set the value to DISABLED in cca_config_set_pkey_mode when NO_PKEY is defined. 

config file description is added.

Reply:
And please add code to not accept anything else than DISABLED if NO_PKEY is defined. We don't want that user think they have it enabled, but it is ignored.